docs/mkdocs.yml

```diff
@@ -666,6 +666,7 @@ nav:
       - 'Deploy a Microsoft Teams personal app and add it to users': 'sample-scripts/teams/install-personal-app.md'
       - 'Deploy Microsoft Teams app from Azure DevOps': 'sample-scripts/teams/deploy-teams-app.md'
       - 'Export conversations from Microsoft Teams Channels': 'sample-scripts/teams/export-teams-conversations.md'
+      - 'Export all channels from Microsoft Teams Team': 'sample-scripts/teams/export-all-channels-teams.md'
```

Comment:
When adding items to the Table of contents, let's sort them alphabetically so that our users can find them more easily.